projects/media-viewer/src/lib/toolbar/toolbar.module.ts

```diff
@@ -3,29 +3,25 @@ import { CommonModule } from '@angular/common';
 import { MainToolbarComponent } from './main-toolbar/main-toolbar.component';
 import { SearchBarComponent } from './search-bar/search-bar.component';
 import { SubToolbarComponent } from './sub-toolbar/sub-toolbar.component';
-import { ToolbarLeftPaneComponent } from './left-pane/left-pane.component';
-import { ToolbarRightPaneComponent } from './right-pane/right-pane.component';
-import { ToolbarMiddlePaneComponent } from './middle-pane/middle-pane.component';
 import { FormsModule } from '@angular/forms';
 import { ToolbarButtonVisibilityService } from './toolbar-button-visibility.service';
 import { ToolbarEventService } from './toolbar-event.service';
 import { RedactionToolbarComponent } from './redaction-toolbar/redaction-toolbar.component';
- import { IcpToolbarComponent } from './icp-toolbar/icp-toolbar.component';
-
+import { IcpToolbarComponent } from './icp-toolbar/icp-toolbar.component';
+import { MediaViewerToolbarComponent } from './media-viewer-toolbar/media-viewer-toolbar.component';
```

Comment:
as mentioned in another comment, this should probably be combined with the main-toolbar component, which can be split into two, one for the horizontal-view and one for the vertical-viewe

Reply:
Agreed. This will be combined with the main-toolbar in the next commit.